### PR TITLE
Feature/what you can enter popup

### LINKED ISSUE #73: [Feature Request] "What you can enter" popup
# Overview

Add a hoverable popup to show what types of searches can be performed.

# Proposal

UTD Grades has this functionality on their home page to quickly show users how to use the site. We need a similar small tutorial that also briefly explains how our data is aggregated.

Below is a screenshot of what that looks like on UTD Grades.

## Supplementary Information

image

### PR DESCRIPTION
**Pull request by TyHil**
## Overview

Resolves #73 

Added the text and popup box to the homepage. 

## What Changed

Just added a MUI Tooltip on some text. Moved the search bar up a tad to accommodate.
This will need a bit of work to work with the dark mode PR.

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -50,11 +66,46 @@ const Home: NextPage = () => {
...
             <Card className="bg-light relative sm:w-5/12 overflow-visible drop-shadow-lg rounded-xl bg-opacity-50 m-auto xs:w-11/12">
-              <div className="bottom-0 absolute text-dark w-full h-1/4 m-auto pb-12 mt-4 mb-4">
+              <div className="bottom-0 absolute text-dark w-full h-1/4 m-auto pb-12 mt-4 mb-6">
                 <SplashPageSearchBar
                   selectSearchValue={searchOptionChosen}
                   disabled={false}
                 />
+                <TransparentTooltip
+                  title={
+                    <Card className="px-3 py-2" elevation={darkModeElevation}>
```
Comment (ZzRanger) [inline pages/index.tsx:76]: What is the purpose of the "elevation" prop?

File: pages/index.tsx [hunk 2]
```diff
@@ -1,19 +1,32 @@
 import type { NextPage } from 'next';
 import Head from 'next/head';
 import { useRouter } from 'next/router';
-import Card from '@mui/material/Card';
+import { Card, Tooltip, Typography, Popper } from '@mui/material';
```
Comment (ZzRanger) [inline pages/index.tsx:4]: Removed unused import `Popper`

File: pages/index.tsx [hunk 3]
```diff
@@ -1,19 +1,32 @@
...
+import { styled } from '@mui/material/styles';
+import { TooltipProps, tooltipClasses } from '@mui/material/Tooltip';
 import { SplashPageSearchBar } from '../components/common/SplashPageSearchBar/splashPageSearchBar';
 import { WaveSVG } from '../components/icons/Wave/waveSVG';
 import { Wave2SVG } from '../components/icons/Wave2/wave2SVG';
 import { LogoIcon } from '../components/icons/LogoIcon/logoIcon';
 import { useState } from 'react';
```
Comment (ZzRanger) [inline pages/index.tsx:11]: Remove unused import

File: pages/index.tsx [hunk 4]
```diff
@@ -30,6 +43,9 @@ const Home: NextPage = () => {
     );
   }
 
+  const prefersDarkMode = useMediaQuery('(prefers-color-scheme: dark)');
+  const darkModeElevation = prefersDarkMode ? 3 : 1;
```
Comment (ZzRanger) [inline pages/index.tsx:47]: AFAIK Trends doesn't support dark mode yet. What is the purpose of this?

**Review by ZzRanger — CHANGES_REQUESTED** [review]
Looks solid! Just a few nits and questions

File: pages/index.tsx (hunk 1 above)
Comment (TyHil) [reply pages/index.tsx:76]: elevation controls the amount of box shadow around the Card. Larger elevation means larger shadow

File: pages/index.tsx (hunk 4 above)
Comment (TyHil) [reply pages/index.tsx:47]: I have another pending PR for dark mode. The change in elevation doesn't add much visual difference in light mode but without it in dark mode it's hard to tell where the popup ends so I thought I'd add this in to comply with the dark mode feature once implemented.

File: pages/index.tsx (hunk 3 above)
Comment (TyHil) [reply pages/index.tsx:11]: Is this also in reference to `Popper`?

File: pages/index.tsx (hunk 2 above)
Comment (TyHil) [reply pages/index.tsx]: Resolved in new commit

File: pages/index.tsx (hunk 3 above)
Comment (ZzRanger) [reply pages/index.tsx:11]: Yeye

**Review by ZzRanger — APPROVED** [review]
Resolve some merge conflicts but after that looks good!

**Pull request merged**